starters/deno-oak-denodb/src/graphql/resolvers/query_handler.ts

```diff
@@ -1,11 +1,34 @@
+import { Cache } from '../../cache/cache.ts';
 import { Technologies } from '../../db/model/technology.ts';
 
-export const getTechnologies = async () => {
-	const technologies = await Technologies.all();
-	return technologies;
+export const getTechnologies = async (
+	_p: unknown,
```

Comment:
Is this a deno specific syntax for using `_p` and `_` as parameter names? 🤔  What's the reason for not having something descriptive?

Reply:
Adding an underscore in front of the variable name tells deno lint that it is an unused variable that is needed due to positional argument